### DISCUSSION TITLE
企业微信推送问题

### DISCUSSION THREAD

**Discussion by tenkawataiyo** [qd-today/qd#266] [Q&A] [answered]
企业微信签到推送，之前几个月都是可以正常推送的，今天突然发现推送已经断在了十天前。
进后台看日志发现签到是正常在签的，只是推送没有推。
测试推送也收不到，检查了各种ID填写，确认是没有错的，甚至重新创建了一个企业微信应用也还是收不到。
请问是哪里出了问题？

**Comment by a76yyyy** [comment, marked as answer]
推送图片cdn链接崩了,已更换,更新框架源码或等待今天docker容器版本推送

**Comment by gklove2008** [comment]
2022525版企业微信还不好用

**Reply by a76yyyy** [reply]
请说明具体细节

**Reply by gklove2008** [reply]
和上面的一样情况，企业微信的各种ID检查好多次，应用重新创建了一个，就是收不到推送。青龙、PT助手什么的都能收到推送，也不知道是哪出问题了。

**Comment by tenkawataiyo** [comment]
6月30号后的推送又不推了，签到正常
